### PR TITLE
Add `get_sweeps` and `delete_sweep` to `SetupHFSS`

### PR DESCRIPTION
**Pull request by atlanswer**
Add two methods `get_sweeps` and `delete_sweep` to the `SetupHFSS` class according to the scripting guide.

Related tests are included.

### CODE REVIEW

File: pyaedt/modules/SolveSetup.py [hunk 1]
```diff
@@ -2542,6 +2542,65 @@ def get_sweep(self, sweepname=None):
...
         return False
 
+    @pyaedt_function_handler()
+    def get_sweeps(self):
+        """Get the names of all sweeps in a given analysis setup.
+
+        Returns
+        -------
+        tuple of str
+            Array of sweep names.
```
Comment (MaxJPRey) [inline pyaedt/modules/SolveSetup.py:2552]:
```suggestion
            Tupple of sweep names.
```

File: pyaedt/modules/SolveSetup.py [hunk 2]
```diff
@@ -2542,6 +2542,65 @@ def get_sweep(self, sweepname=None):
...
+
+        References
+        ----------
+
+        >>> oModules.GetSweeps
+
+        Examples
+        --------
+        >>> hfss = Hfss()
```
Comment (MaxJPRey) [inline pyaedt/modules/SolveSetup.py:2561]:
```suggestion
        >>> import pyaedt
        >>> hfss = pyaedt.Hfss()
```

File: pyaedt/modules/SolveSetup.py [hunk 3]
```diff
@@ -2542,6 +2542,66 @@ def get_sweep(self, sweepname=None):
                 return self.sweeps[0]
         return False
 
+    @pyaedt_function_handler()
+    def get_sweeps(self):
```
Comment (Samuelopez-ansys) [inline pyaedt/modules/SolveSetup.py:2546]:
```suggestion
    def get_sweep_names(self):
```
I would change the name to have naming consistency with other PyAEDT methods

**Review by Samuelopez-ansys — APPROVED** [review]

File: pyaedt/modules/SolveSetup.py [hunk 4]
```diff
@@ -2542,6 +2542,66 @@ def get_sweep(self, sweepname=None):
...
+        ----------
+
+        >>> oModules.GetSweeps
+
+        Examples
+        --------
+        >>> import pyaedt
+        >>> hfss = pyaedt.Hfss()
+        >>> setup = hfss.get_setup('Pyaedt_setup')
+        >>> sweeps = setup.get_sweeps()
```
Comment (Samuelopez-ansys) [inline pyaedt/modules/SolveSetup.py:2564]:
```suggestion
        >>> sweeps = setup.get_sweep_names()
```

File: pyaedt/modules/SolveSetup.py [hunk 5]
```diff
@@ -2542,6 +2542,66 @@ def get_sweep(self, sweepname=None):
...
+
+        >>> import pyaedt
+        >>> hfss = pyaedt.Hfss()
+        >>> setup1 = hfss.create_setup(setupname='Setup1')
+        >>> setup1.create_frequency_sweep(
+            "GHz", 24, 24.25, 26, "Sweep1", sweep_type="Fast",
+        )
+        >>> setup1.delete_sweep("Sweep1")
+        """
+        if sweepname in self.get_sweeps():
```
Comment (Samuelopez-ansys) [inline pyaedt/modules/SolveSetup.py:2599]:
```suggestion
        if sweepname in self.get_sweep_names():
```

File: _unittest/test_11_Setup.py [hunk 6]
```diff
@@ -114,6 +114,16 @@ def test_05_sweep_auto(self):
         assert setup1.add_subrange("LinearStep", 1, 10, 0.1, clear=False)
         assert setup1.add_subrange("LinearCount", 10, 20, 10, clear=True)
 
+    def test_05a_delete_sweep(self):
+        setup1 = self.aedtapp.create_setup("My_HFSS_Setup5", self.aedtapp.SETUPS.HFSSDrivenDefault)
+        setup1.create_frequency_sweep("GHz", 24, 24.25, 26, "My_Sweep1", sweep_type="Fast")
+        sweeps = setup1.get_sweeps()
```
Comment (Samuelopez-ansys) [inline _unittest/test_11_Setup.py:120]:
```suggestion
        sweeps = setup1.get_sweep_names()
```

**Review by Samuelopez-ansys — CHANGES_REQUESTED** [review]

**Comment by Samuelopez-ansys** [comment]
@atlanswer Please check my comments. Did you check if somewhere else the sweep is still available after you delete it? I will review it later on.

**Comment by atlanswer** [comment]
@Samuelopez-ansys Thanks. I think sweeps are stored in `SetupHFSS.sweeps` and in AEDT. Both are deleted here.

**Review by Samuelopez-ansys — APPROVED** [review]
Great enhacement!

**Pull request merged**